### PR TITLE
feat(Core/DB): Align DB column names to TrinityCore

### LINKED ISSUE #11005: Feature: Align DB column names to TrinityCore
### Describe your feature request or suggestion in detail

In line with https://github.com/azerothcore/azerothcore-wotlk/issues/1118, there's a few column discrepancies between azerothcore and trinitycore that come down to just different naming conventions. 

I've been asked to post this issue as a step to bringing AzerothCore closer to the TSWoW project. For our sake, these specific changes are **not really necessary anymore** as we currently [apply a translation layer between ac and tc](https://github.com/tswow/tswow/commit/b52b5877a580f64b5b6c7a5e6b54b5f1d4987bfb) covering all these fields, but I thought you might find our comparison scripts useful for other tools and such.

### Describe a possible solution to your feature or suggestion in detail

The following are columns that just have different names in AzerothCore and TrinityCore. In my opinion, for some of these your names are better so I wouldn't really suggest changing those. Maybe we can convince trinity to change theirs in those instances.

### broadcast_text
`MaleText` -> `Text`
`FemaleText` -> `Text1`
`SoundEntriesId` -> `SoundEntriesID`

### broadcast_text_locale
`MaleText` -> `Text`
`FemaleText` -> `Text1`

### game_event_battleground_holiday
`eventEntry` -> `EventEntry`

### npc_text
(for x 0-7 and y 0-5):
`em{x}_{y}` -> `Emote{x}_{y}`

### player_factionchange_items
`alliance_comment` -> `commentA`
`horde_comment` -> `commentH`

### spell_enchant_proc_data
`entry` -> `EnchantID`
`customChance` -> `Chance`
`PPMChance` -> `ProcsPerMinute`
`procEx` -> `AttributesMask`

### spell_proc
`spellId` -> `SpellId`
 `schoolMask`  -> `SchoolMask`
`spellFamilyName` -> `SpellFamilyName`
`spellFamilyMask0` -> `SpellFamilyMask0`
`spellFamilyMask1` -> `SpellFamilyMask1`
`spellFamilyMask2` -> `SpellFamilyMask2`
`spellTypeMask` -> `SpellTypeMask`
`spellPhaseMask` -> `SpellPhaseMask`
`hitMask` -> `HitMask`
`attributesMask` -> `AttributesMask`
`ratePerMinute` -> `RatePerMinute`
`chance` -> `Chance`
`cooldown` -> `Cooldown`
`charges` -> `Charges`

### achievement_reward_locale
`Text` -> `Body`

### AC rev. hash/commit
https://github.com/tswow/azerothcore-wotlk/commit/ee442e2f571eba480e92418c9fce32e723ca0973

### Additional context

Here is the [script i used to find these differences](https://gist.github.com/ihm-tswow/8700eda73ca0d33fd88b42874163c91c). It can detect missing tables, missing columns and type discrepancies between columns. Note that it does not detect different column orders, as we didn't need it for our application. You can modify the script to enable this if you need it.

To run the comparator, take the following steps:

1. Place the `dbccomparator.js` script in an empty directory
2. Edit the `DB_1` and `DB_2` variables to the names of your trinitycore/azerothcore database names.
3. In the directory where you placed the script, run `npm i mysql` to install the adapter
4. In the same directory, run `node dbcomparator` to compare the databases.
5. The output will show you missing tables, columns and type discrepancies between existing columns.

<bountysource-plugin>

---
Want to back this issue? **[Post a bounty on it!](https://app.bountysource.com/issues/106210514-feature-align-db-column-names-to-trinitycore?utm_campaign=plugin&utm_content=tracker%2F40032087&utm_medium=issues&utm_source=github)** We accept bounties via [Bountysource](https://app.bountysource.com/?utm_campaign=plugin&utm_content=tracker%2F40032087&utm_medium=issues&utm_source=github).
</bountysource-plugin>

### PR DESCRIPTION
**Pull request by malcrom**
## Changes Proposed:
- Rename some Azerothcore DB columns to match Trinitycore.

## Issues Addressed:
- Closes #11005

## SOURCE:
[ihm-tswow](https://github.com/ihm-tswow)

## Tests Performed:
- Compiles
- No DB errors
- No startup errors

### CODE REVIEW

File: data/sql/updates/pending_db_world/rev_1648063145113494892.sql [hunk 1]
```diff
@@ -0,0 +1,78 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1648063145113494892');
+
+ALTER TABLE `broadcast_text` CHANGE `MaleText` `Text` LONGTEXT;
+ALTER TABLE `broadcast_text` CHANGE `FemaleText` `Text1` LONGTEXT;
```
Comment (Bogir) [inline data/sql/updates/pending_db_world/rev_1648063145113494892.sql:4]: We need to add an explanation that `Text` is MaleText and `Text1` is FemaleText 

```
ALTER TABLE `broadcast_text` CHANGE `MaleText` `Text` LONGTEXT COMMENT 'MaleText';
ALTER TABLE `broadcast_text` CHANGE `FemaleText` `Text1` LONGTEXT COMMENT 'FemaleText';
```
```
ALTER TABLE `broadcast_text_locale` CHANGE `MaleText` `Text` LONGTEXT COMMENT 'MaleText';
ALTER TABLE `broadcast_text_locale` CHANGE `FemaleText` `Text1` LONGTEXT COMMENT 'FemaleText';
```
https://www.azerothcore.org/wiki/broadcast_text#broadcast_text-MaleText

изображение

Comment (malcrom) [reply data/sql/updates/pending_db_world/rev_1648063145113494892.sql]: Updated

**Comment by Lynethris** [comment]
Status ? Need test ?

**Review by Kitzunu — APPROVED** [review]

**Comment by Kitzunu** [comment]
@malcrom build errors

**Comment by acidmanifesto** [comment]
This PR is closed per Private message request from Malcrom.

**Pull request closed without merging**